apps/finance/app/src/components/Transfers.js

```diff
@@ -199,13 +202,28 @@ const Transfers = React.memo(({ tokens, transactions }) => {
     [filteredTransfers, compareDesc]
   )
 
-  if (!transactions.length) {
-    return <EmptyTransactions />
-  }
+  const dataViewStatus = useMemo(() => {
+    if (emptyResultsViaFilters && transactions.length > 0) {
```

Comment:
Here I noticed we actually have to check for the second condition, because if not, when the DataView is empty we'll get a non-accurate "empty-filters" screen, which is not right.

Reply:
Ah yes you’re right!